### DISCUSSION TITLE
More Z Seam Options

### DISCUSSION THREAD

**Discussion by tommy536** [OrcaSlicer/OrcaSlicer#416] [Ideas]
Hello!

I am using this great fork with a Voron 2.4. Installed is the Stealthburner. In some cases or overhangs to the back, the left and right airstream isnt touching the right place. I can turn the object but then, the seam is on the front of the object which didnt look good. 

Please add some more positions if possible. Seam on the Left, Right, or Front would be nice.

**Comment by KriswiffaK** [comment]
Why not just draw the seams around your problem area
